cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureNetworkConnector.java

```diff
@@ -137,6 +139,16 @@ public String getNetworkCidr(Network network, CloudCredential credential) {
         return networkCidrs.get(0);
     }
 
+    @Override
+    public List<CloudSubnet> selectSubnets(List<CloudSubnet> subnetMetas, SubnetSelectionParameters subnetSelectionParameters) {
+        if (subnetMetas.isEmpty()) {
```

Comment:
on AWS implementation there is a validation if `subnetMetas` is null. I think it would make sense to have the same here

Reply:
Yes, thanks, I added it.